pkg/metrics/resource_metrics_collector.go

```diff
@@ -0,0 +1,87 @@
+package metrics
+
+import (
+	"sync"
+	"sync/atomic"
+	"time"
+
+	"github.com/gocrane/crane/pkg/utils"
+	"github.com/prometheus/client_golang/prometheus"
+	"k8s.io/component-base/metrics"
+	"k8s.io/component-base/metrics/legacyregistry"
+)
+
+var (
+	nodeExtCPUUsageDesc = prometheus.NewDesc("node_ext_cpu_usage_seconds_total",
+		"Cumulative cpu time consumed by the node in core-seconds",
+		[]string{"node"},
+		nil)
+	nodeCPUCanBeReusedDesc = prometheus.NewDesc("node_cpu_can_be_reused_seconds",
+		"Cumulative cpu time consumed by the node in core-seconds",
```

Comment:
Cumulative cpu time consumed by the node in core-seconds
->
Reallocated CPU consumed by the node in cores